piksi_tools/bootload.py

```diff
@@ -81,7 +81,7 @@ def handshake(self, timeout=None):
       t0 = time.time()
     self.handshake_received = False
     expire = time.time() + 15.0
-    self.link.send(SBP_MSG_RESET, "")
+    self.link(MsgReset())
```

Comment:
Hmm.  There was, but it disappeared in my work on the iterators and isn't in 0.50.1.  This will have to do for now.